pydantic_ai_slim/pydantic_ai/models/anthropic.py

```diff
@@ -151,28 +152,34 @@ class AnthropicAgentModel(AgentModel):
     allow_text_result: bool
     tools: list[ToolParam]
 
-    async def request(self, messages: list[Message]) -> tuple[ModelAnyResponse, result.Cost]:
-        response = await self._messages_create(messages, False)
+    async def request(
+        self, messages: list[Message], model_settings: ModelSettings | None = None
+    ) -> tuple[ModelAnyResponse, result.Cost]:
+        response = await self._messages_create(messages, False, model_settings)
         return self._process_response(response), _map_cost(response)
 
     @asynccontextmanager
-    async def request_stream(self, messages: list[Message]) -> AsyncIterator[EitherStreamedResponse]:
-        response = await self._messages_create(messages, True)
+    async def request_stream(
+        self, messages: list[Message], model_settings: ModelSettings | None = None
```

Comment:
```suggestion
        self, messages: list[Message], model_settings: ModelSettings | None
```
and elsewhere.

Reply:
Yep, thanks! Initially had a question about this in the PR description. Removed, since this resolves my inquiry.